File: appointing_desktop/lib/models/doctor.dart
class Doctor {
  Doctor({
    this.doctorId,
    this.name,
    this.fee,
    this.specialization,
    this.opd,
    this.image,
  });

  int doctorId;
  String name;
  String fee;
  String specialization;
  String opd;
  String image;

  factory Doctor.fromJson(Map<String, dynamic> json) => Doctor(
        doctorId: json["doctorId"],
        name: json["name"],
        fee: json["fee"],
        specialization: json["specialization"],
        opd: json["opd"],
        image: json["image"],
      );

  Map<String, dynamic> toJson() => {
        "doctorId": doctorId,
        "name": name,
        "fee": fee,
        "specialization": specialization,
        "opd": opd,
        "image": image,
      };
}

File: appointing-api/src/main/java/com/hospital/management/model/Doctor.java
package com.hospital.management.model;

import java.util.ArrayList;
import java.util.Collection;

import javax.persistence.CascadeType;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Lob;
import javax.persistence.OneToMany;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.hospital.management.relationTable.Appointment;

@Entity
@Table(name = "doctor")
public class Doctor {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer doctorId;

	private String name;
	private String fee;
	private String specialization;
	private String opd;
	@Lob
	private String image;

	@OneToMany(mappedBy = "doctor", cascade = CascadeType.ALL)
	@JsonIgnore
	private Collection<Appointment> appointment = new ArrayList<>();

	public Doctor() {
	}

	public Integer getDoctorId() {
		return doctorId;
	}

	public void setDoctorId(Integer doctorId) {
		this.doctorId = doctorId;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getFee() {
		return fee;
	}

	public void setFee(String fee) {
		this.fee = fee;
	}

	public String getSpecialization() {
		return specialization;
	}

	public void setSpecialization(String specialization) {
		this.specialization = specialization;
	}

	public String getOpd() {
		return opd;
	}

	public void setOpd(String opd) {
		this.opd = opd;
	}

	public Collection<Appointment> getAppointment() {
		return appointment;
	}

	public void setAppointment(Collection<Appointment> appointment) {
		this.appointment = appointment;
	}

	public String getImage() {
		return image;
	}

	public void setImage(String image) {
		this.image = image;
	}

}